src/nomad_simulations/schema_packages/properties/integrals.py

```diff
@@ -0,0 +1,150 @@
+from typing import TYPE_CHECKING
+
+import numpy as np
+from nomad.metainfo import MEnum, Quantity
+
+if TYPE_CHECKING:
+    from nomad.datamodel.datamodel import EntryArchive
+    from nomad.metainfo import Context, Section
+    from structlog.stdlib import BoundLogger
+
+from nomad_simulations.schema_packages.physical_property import PhysicalProperty
+
+
+class OneElectronIntegral(PhysicalProperty):
+    """
+    One-electron matrices **⟨χᵢ| Ô |χⱼ⟩** expressed in either an atomic-orbital (AO)
```

Comment:
Pls double-check that the bra-ket notation renders well in your local instance. Else, you can use Latex formatting instead.